app/controllers/data_services_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class DataServicesController < ApplicationController
   def show
-    @data_service = DataService.find(params[:id])
+    @data_service = DataService.includes(:creators, :publisher, data_resource: :related_resources).find(params[:id])
```

Comment:
With the `has_many through` you should now also be able to just do `:related_resources` without needing to go through the data resource :)

Reply:
Fab, I've just added a commit with it removed :)